cmd/loki/loki-local-config.yaml

```diff
@@ -31,37 +29,10 @@ schema_config:
         prefix: index_
         period: 24h
 
-storage_config:
-  boltdb_shipper:
-    active_index_directory: /tmp/loki/boltdb-shipper-active
-    cache_location: /tmp/loki/boltdb-shipper-cache
-    cache_ttl: 24h         # Can be increased for faster performance over longer query periods, uses more disk space
-    shared_store: filesystem
-  filesystem:
-    directory: /tmp/loki/chunks
-
-compactor:
-  working_directory: /tmp/loki/boltdb-shipper-compactor
-  shared_store: filesystem
-
-limits_config:
-  unordered_writes: true
-
-chunk_store_config:
-  max_look_back_period: 0s
-
-table_manager:
-  retention_deletes_enabled: false
-  retention_period: 0s
-
 ruler:
   storage:
     type: local
-    local:
-      directory: /tmp/loki/rules
-  rule_path: /tmp/loki/rules-temp
   alertmanager_url: http://localhost:9093
   ring:
     kvstore:
       store: inmemory
-  enable_api: true
```

Comment:
Hmm the default `enable_api` defined in cortex/ruler is actually false. But I think it is fine to not have an API for this more-simple config.

Reply:
Nope, it's true now: https://github.com/grafana/loki/blob/main/pkg/ruler/config.go#L34